permissions/rds-protection/1017.json

```diff
@@ -0,0 +1,445 @@
+            "Assigning tags to an RDS resource."
+          ]
+        },
+        {
+          "Permission": "rds:RemoveTagsFromResource",
+          "UseCases": [
+            "Removing tags to an RDS resource."
+          ]
+        },
+        {
+          "Permission": "rds:ListTagsForResource",
+          "UseCases": [
+            "Listing tags for an RDS resource."
+          ]
+        },
+        {
+          "Permission": "rds:DescribeDBParameterGroups",
+          "UseCases": [
+            "Listing database instance parameter groups for export."
+          ]
+        },
+        {
+          "Permission": "rds:DescribeDBClusterParameterGroups",
+          "UseCases": [
+            "Listing database cluster parameter groups for export."
+          ]
+        },
+        {
+          "Permission": "rds:DescribeDBSubnetGroups",
+          "UseCases": [
+            "Listing DB subnet groups for export."
+          ]
+        },
+        {
+          "Permission": "rds:DescribeOptionGroups",
+          "UseCases": [
+            "Listing database instance option groups for export."
+          ]
+        },
+        {
+          "Permission": "rds:DescribeOrderableDBInstanceOptions",
+          "UseCases": [
+            "Listing valid instance types for export.",
+            "Validating parameters for export."
+          ]
+        },
+        {
+          "Permission": "rds:DescribeDBInstanceAutomatedBackups"
+        },
+        {
+          "Permission": "rds:RestoreDBInstanceFromDBSnapshot",
+          "UseCases": [
+            "Exporting database instance snapshots during recovery."
+          ]
+        },
+        {
+          "Permission": "rds:RestoreDBInstanceToPointInTime",
+          "UseCases": [
+            "Used for point-in-time recovery of a database instance."
+          ]
+        },
+        {
+          "Permission": "rds:CreateDBSubnetGroup"
+        },
+        {
+          "Permission": "rds:DeleteDBSubnetGroup"
+        },
+        {
+          "Permission": "rds:CopyDBSnapshot",
+          "UseCases": [
+            "Replicate snapshots to other accounts and regions."
+          ]
+        },
+        {
+          "Permission": "rds:DescribeDBEngineVersions",
+          "UseCases": [
+            "Retrieving information about DB engine version."
+          ]
+        },
+        {
+          "Permission": "rds:ModifyDBSnapshotAttribute",
+          "UseCases": [
+            "Replicate snapshots to other accounts and regions."
+          ]
+        },
+        {
+          "Permission": "rds:ModifyDBClusterSnapshotAttribute",
+          "UseCases": [
+            "Replicate snapshots of a database cluster to other accounts and regions."
+          ]
+        }
+      ],
+      "Resource": [
+        "*"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": [
+        {
+          "Permission": "cloudwatch:GetMetricStatistics",
+          "UseCases": [
+            "Fetching usage metrics."
```

Comment:
Should we add more details on which metrics we are fetching, since this is for customer reference?

Reply:
This should be good enough. This is not customer facing anymore.